solutions/search/vector/knn.md

```diff
@@ -889,6 +889,169 @@ Now the result will contain the nearest found paragraph when searching.
 }
 ```
 
+### Customize nested kNN search
+
+You can combine nested kNN search with top-level dense vector fields, filters, and `inner_hits` to support more advanced use cases. 
+
+This approach is helpful when you:
```

Comment:
The biggest use case for using `nested` for `knn` searches is chunking, it would be nice to call it out here.